react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+const images = ['001', '004', '007', '025', '039'];
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      interval: setInterval(() => this.nextImage(), 3000),
```

Comment:
Starting your initial interval is best done in `componentDidMount`

https://reactjs.org/docs/react-component.html#componentdidmount